File: src/main/java/com/baulsupp/oksocial/services/stackexchange/StackExchangeToken.java
package com.baulsupp.oksocial.services.stackexchange;

public class StackExchangeToken {
  public String accessToken;
  public String key;

  public StackExchangeToken() {
  }

  public StackExchangeToken(String accessToken, String key) {
    this.accessToken = accessToken;
    this.key = key;
  }
}

File: src/main/kotlin/com/baulsupp/oksocial/services/stackexchange/StackExchangeToken.kt
package com.baulsupp.oksocial.services.stackexchange

class StackExchangeToken {
    var accessToken: String
    var key: String

    constructor() {}

    constructor(accessToken: String, key: String) {
        this.accessToken = accessToken
        this.key = key
    }
}
